### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by RWUBAKWANAYO**
# In this project we:

- [ ] Create a `app.rb` file that will serve as console app entry-point. by:
- List all books.
- List all people.
- Create a person (teacher or student, not a plain `Person`).
- Create a book.
- Create a rental.
- List all rentals for a given person id.

- [ ] define the entry point, in `main.rb` file which is a method called `main` that is invoked at the end of the file. The method do the following:
- Present the user with a list of options to perform.
- Lets users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app.

***

**N.B: `output.rb` file used to test the previous tasks, to interact with UI, we should run  `ruby main.rb`  command.**

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      create_book(books)
+    when 5
+      create_rental(books, people, rentals)
+    when 6
+      list_all_rentals_by_id(rentals)
+    else
+      break
+    end
+  end
+end
```
Comment (rustam-tolipov) [inline app.rb]: - [ ] Please put these methods into main.rb to make app.rb file more slimmer.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @RWUBAKWANAYO, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @RWUBAKWANAYO 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by RWUBAKWANAYO** [comment]
merge changes to the dev branch

**Pull request merged**